hooli_data_eng/jobs/watch_s3.py

```diff
@@ -31,51 +39,31 @@ def get_env():
         return "PROD"
     return "LOCAL"
 
-sensor_resources = {
-    "LOCAL": {"fs": local_fs.configured({
-        "base_dir": file_relative_path(__file__, ".")
-    })},
-    "PROD": {"fs": s3_fs.configured({
-        "region_name": "us-west-2",
-        "s3_bucket": "hooli-demo"
-    })},
-    "BRANCH": {"fs": s3_fs.configured({
-        "region_name": "us-west-2",
-        "s3_bucket": "hooli-demo-branch"
-    })}
-}
-
 
 # This sensor watches for updates to the file
 # "customers.txt"
-@sensor(
-    job=log_file_job
-)
-def watch_s3_sensor(context):
+@sensor(job=log_file_job)
+def watch_s3_sensor(context, monitor_fs: FileSystem):
```

Comment:
Here, the argument name must match the name of a resource provided to `Definitions` (here `monitor_fs`) and be annotated with `Resource[...]` or something extending `ConfigurableResource` (here `FileSystem` extends `ConfigurableResource`).